common/ta_errors.h

```diff
@@ -169,6 +169,10 @@ typedef enum {
   /**< fail to initialize yaml parser */
   SC_CONF_FOPEN_ERROR = 0x07 | SC_MODULE_CONF | SC_SEVERITY_FATAL,
   /**< fail to open file */
+  SC_CONF_IRI_BEHIND = 0x08 | SC_MODULE_CONF | SC_SEVERITY_MAJOR,
```

Comment:
How about changing `SC_CONF_IRI_BEHIND` to `SC_CONF_IRI_UNSYNC`?

Reply:
actually I am thinking should we create a new category which named as `SC_MODULE_CORE`?

Reply:
yeah, that will be more precise.

Reply:
So the status code would be `SC_CORE_IRI_UNSYNC`. What do you think